osquery/killswitch/killswitch_plugin.cpp

```diff
@@ -0,0 +1,59 @@
+#include <string>
+
+#include <chrono>
+#include <osquery/flags.h>
+#include <osquery/killswitch.h>
+#include <osquery/killswitch/killswitch_plugin.h>
+#include <osquery/registry_factory.h>
+
+namespace osquery {
+
+CREATE_REGISTRY(KillswitchPlugin, "killswitch");
+
+Status KillswitchPlugin::call(const PluginRequest& request,
+                              PluginResponse& response) {
+  auto action = request.find("action");
+  if (action == request.end()) {
+    return Status(1, "Killswitch plugins require an action");
+  }
+  if (action->second == "isEnabled") {
```

Comment:
There are a lot of different string constants in the code it means a lot of place to misspell something. And such misspell will not be cause of compilation error. Instead of it it will cause of some runtime problem. So, to avoid it I'd suggest to define a proper constants for this literals and use literal consts instead of hardcoded strings. What do you think?